aws/aws.go

```diff
@@ -253,6 +253,48 @@ func AllInstances() chan *Instance {
 	return ch
 }
 
+// AllVolumes describes every instance in the requested regions
+// *Volumes are created for each *ec2.Volume
+// and are passed to a channel
+func AllVolumes() chan *Volume {
+	ch := make(chan *Volume)
```

Comment:
Since the length of `config.Regions` is known, the channel could be made with `make(chan *Volume, len(config.Regions))`, so none of the goroutines would be blocked.